src/content/configuration/other-options.md

```diff
@@ -146,3 +153,17 @@ module.exports = {
   recordsOutputPath: path.join(__dirname, 'newRecords.json')
 };
 ```
+
+
+## `name`
+
+`string`
+
```

Comment:
please also add `__webpack.config.js__` above the code blocks

Reply:
Both review fixes done. Also fixed inconsistency in other examples on this page.